src/commands/pub.ts

```diff
@@ -36,7 +36,7 @@ export class PubManager {
 		// return the paths for tools in the bin/ dir. 
 		let pubPath = path.join(this.sdk, "bin", "pub");
 		channel.appendLine(`[${shortPath}] pub ${command}`);
-		let process = child_process.exec(`${pubPath} ${command}`, { "cwd": projectPath });
```

Comment:
Did some testing on Raspbian, and I think just escaping the quotes with blackslash will resolve this:

```
`"${pubPath.replace(/"/g, "\\\""")}" ${command}`
```